src/blocks/bluetooth.rs

```diff
@@ -75,44 +75,36 @@ pub async fn run(config: toml::Value, mut api: CommonApi) -> Result<()> {
             // Available
             Some(device) => {
                 let connected = device.connected().await?;
-                if connected || !config.hide_disconnected {
-                    let mut values = map! {
-                        "name" => Value::text(device.name().await?),
-                        "available" => Value::flag(),
-                    };
-                    device
-                        .percentage()
-                        .await
-                        .map(|p| values.insert("percentage".into(), Value::percents(p)));
-                    if connected {
-                        widget.state = State::Good;
-                        values.insert("connected".into(), Value::flag());
-                        debug!("Showing device as connected");
-                    } else {
-                        debug!("Showing device as disconnected");
-                        widget.state = State::Idle;
-                    }
-                    widget.set_icon(device.icon().await?)?;
-                    widget.set_values(values);
-
-                    api.set_widget(&widget).await?;
+                let mut values = map! {
+                    "icon" => Value::icon(api.get_icon(device.icon().await?)?),
+                    "name" => Value::text(device.name().await?),
+                    "available" => Value::flag(),
```

Comment:
Not sure if this flag makes sense anymore

Reply:
It's still needed. I wasn't aware how to fully nicely test this but I've found this config: https://sourcegraph.com/github.com/ammgws/dotfiles/-/blob/i3status-rs/.config/i3status-rust/config.toml?L205&subtree=true using it :smile: 

```toml
format = "$connected$available{PS5}|$available{D/C}|N/A"
```

On master I got:

```json
[
    // unknown
    {"full_text":"  ","name":"0"},
    {"full_text":"N/A ","name":"0"},
    // connected
    {"full_text":"  ","name":"1"},
    {"full_text":"PS5 ","name":"1"}
    // known, diconnected
    {"full_text":"  ","name":"1"},
    {"full_text":"D/C","name":"1"}
  ]
```

To fully repro the same behaviour I've used:
```toml
format = " $icon PS5 "
disconnected_format = " $icon$available {D/C} | $icon N/A "
```
This gives me:

```json
  [

    // unknown
    {"full_text":"  N/A ","name":"0"},
    // connected
    {"full_text":"  PS5 ", "name": 1}
    // known, disconnected
    {"full_text":"  ","name":"1"},
    {"full_text":"D/C ","name":"1"}
  ]
```

Reply:
Yeah, but I though maybe unavailable can be treated just like disconnected. Similar how `net` treats missing and down interfaces the same way.

Reply:
I think we have to agree on the same convention and naming :smile: `disconnected_format` vs `missing_format` naming wise and then if `$available` should be added when the device exists vs adding another format option. From my understanding, that's the only way to make the old behavior possible...

Reply:
> `disconnected_format` vs `missing_format`

"missing" is shorter but not quite correct :laughing: "disconnected" seems better to me.

> `format = "$connected$available{PS5}|$available{D/C}|N/A"`

@ammgws Do you rely on the `N/A` part of this format? What are you thoughts?

Reply:
I think I had that in since at one stage my USB bluetooth adapter was at times not plugged in and I wanted to distinguish between the device being completely unavailable (nothing on dbus) vs the device actually just being in a disconnected state.

Reply:
Okay, so it has it's usecases. Perhaps it's better to add `unavailable_format`? I don't know how I feel about this whole "flags" idea.

Reply:
Would it be ok for you to tackle this on another PR and leave the flag for now as it is? I think going 100% with "foobar_format" is interesting and would be more explicit.

Reply:
> Would it be ok for you to tackle this on another PR and leave the flag for now as it is?

Yeah, sure!